ovs-driver/src/main/java/org/opendaylight/unimgr/mef/nrp/ovs/util/OpenFlowUtils.java

```diff
@@ -138,13 +139,19 @@ public static Table getTable(Node node) throws ResourceNotAvailableException {
     }
 
     /**
-     * Returns list of flows installed in flow table
+     * Returns list of flows installed in flow table. Flows which match LLDP packets are filtered out.
      *
      * @param table flow table
      * @return list of flows
      */
-    public static List<Flow> getExistingFlows(Table table) {
-        return table.getFlow().stream().map(x -> new FlowBuilder(x).build()).collect(Collectors.toList());
+    public static List<Flow> getExistingFlowsWithoutLLDP(Table table) {
+        return table.getFlow().stream().map(x -> new FlowBuilder(x).build()).
+                filter(flow -> {
+                    if (flow.getMatch().getEthernetMatch() == null) return true;
+                    if (flow.getMatch().getEthernetMatch().getEthernetType().getType().getValue().equals(Long.valueOf(EtherTypes.LLDP.intValue()))) return false;
```

Comment:
it can still result in NPE 
you can try sth like
try {
return ! flow.getMatch().getEthernetMatch().getEthernetType().getType().getValue().equals(Long.valueOf(EtherTypes.LLDP.intValue()))
} catch (NPE e) {}
return true;